docs/02.API-REFERENCE.md

```diff
@@ -4960,6 +5099,37 @@ main (void)
 
 - [jerry_module_link](#jerry_module_link)
 
+
+## jerry_module_cleanup
+
+**Summary**
+
+Release known modules in the current context. If realm parameter is supplied, cleans up modules native to that realm only. This function should be called by the user application when the module database in the current context is no longer needed.
+
+**Prototype**
+
+```c
+void jerry_module_cleanup (const jerry_value_t realm);
+```
+
+- `realm` - release only those modules which realm value is equal to this argument.
```

Comment:
Sorry for all the nitpicking, but you should use 'whose' instead of 'which' because it refers to possession.